src/testbed_nerf.cu

```diff
@@ -2246,7 +2246,6 @@ void Testbed::Nerf::Training::update_transforms(int first, int last) {
 void Testbed::create_empty_nerf_dataset(size_t n_images, Eigen::Vector2i image_resolution, int aabb_scale, bool is_hdr) {
 	m_nerf.training.dataset = ngp::create_empty_nerf_dataset(n_images, image_resolution, aabb_scale, is_hdr);
 	load_nerf();
-	m_nerf.training.n_images_for_training = 0;
```

Comment:
The intention of this line was to prevent learning from the empty images that are created by `ngp::create_empty_nerf_dataset` -- could you revert this change before I hit merge?

A better fix for problems with training on `m_nerf.training.n_images_for_training == 0` images would be to check for this condition in the training function and to turn it into a no-op in that case. I'd be happy to take care of that fix after merging your PR.

Reply:
Sure, I think, maybe it's better to add a counter or something else, to check whether all the images and their camera params are initialized before training. And I have reverted to the previous commit.

Reply:
I also find a simple workaround.
We are able to directly control the variable `n_images_for_training` from python using `testbed.nerf.training.n_images_for_training += 1` after we manually add an image to the empty dataset.